docs/content/preview/benchmark/sysbench-ysql.md

```diff
@@ -111,17 +118,15 @@ $ sysbench <workload>               \
 
 ## Expected results
 
-The following results are for a 3-node cluster with each node on a c5.4xlarge AWS instance (16 cores, 32 GB of RAM, and 2 EBS volumes), all in the same AZ with the client VM running in the same AZ.
+The following results are for a 3-node cluster, with each node running on a c5.2xlarge AWS instance (8 cores, 16 GiB of RAM), all in the same AZ, with a replication factor of 3 and TLS enabled.
```

Comment:
Running what version of YBDB?

Reply:
They didn't mention it in the doc that was given to me. @shaharuk-yb @shantanugupta-yb can provide an input here.

Reply:
The numbers are from build 2024.1.4.0-b84.

Reply:
What about the disk configuration? cc @shantanugupta-yb

Reply:
Lets maintain the numbers per release. We can mention the build that was run.

Reply:
The disk configurations for these tests are set at IOPS: 3000 and throughput 125 MBps per node

Reply:
I've mentioned the specifc version for each release using shortcode